README.md

```diff
@@ -235,6 +235,7 @@ The image has the following extensions pre-installed, there extensions can be en
 * Scribunto
 * SecureLinkFixer
 * SelectCategory
+* SemanticMediaWiki
```

Comment:
Despite the SemanticMediaWiki is bundled with the image it'd be incorrect to have it on this list since the above states that the extensions can be enabled via `MW_LOAD_EXTENSIONS` env, which is not true for the SMW

Maybe let's add an extra list down below?

Reply:
Yep, sounds good.